src/adapter/z-stack/adapter/zStackAdapter.ts

```diff
@@ -221,14 +214,25 @@ class ZStackAdapter extends Adapter {
     }
 
     public async permitJoin(seconds: number, networkAddress?: number): Promise<void> {
-        const addrmode = networkAddress === undefined ? 0x0f : 0x02;
-        const dstaddr = networkAddress || 0xfffc;
+        const clusterId = Zdo.ClusterId.PERMIT_JOINING_REQUEST;
```

Comment:
Note to self: check if this doesn't break https://github.com/Koenkk/Z-Stack-firmware/blob/7398d834eb3a790876c280293c4181da96cc7114/coordinator/Z-Stack_3.x.0/firmware.patch#L274

Reply:
It doesn't, because the native requests are still being used